tests/sequential/prometheus_metrics/prometheus_metrics_test.go

```diff
@@ -292,6 +322,121 @@ func testScalerMetricValue(t *testing.T) {
 	}
 }
 
+func testScaledObjectErrors(t *testing.T, data templateData) {
+	t.Log("--- testing scaled object errors ---")
+
+	KubectlDeleteWithTemplate(t, data, "scaledObjectTemplate", scaledObjectTemplate)
+	KubectlApplyWithTemplate(t, data, "wrongScaledObjectTemplate", wrongScaledObjectTemplate)
+
+	family := fetchAndParsePrometheusMetrics(t, fmt.Sprintf("curl --insecure %s", kedaOperatorPrometheusURL))
+	if val, ok := family["keda_scaled_object_errors"]; ok {
+		errCounterVal1 := getErrorMetricsValue(val)
+
+		// wait for 2 seconds as pollinginterval is 2
+		time.Sleep(2 * time.Second)
+
+		family = fetchAndParsePrometheusMetrics(t, fmt.Sprintf("curl --insecure %s", kedaOperatorPrometheusURL))
+		if val, ok := family["keda_scaled_object_errors"]; ok {
+			errCounterVal2 := getErrorMetricsValue(val)
+			assert.Greater(t, errCounterVal2, errCounterVal1)
```

Comment:
I guess that this assertion should be `equal`, shouldn't? 
As now the test is executed sequentially, you can control all the values (there isn't any other test in progress at the same time). 
To be sure about the previous errors, you can delete KEDA pods during the test preparation ([this is an example](https://github.com/kedacore/keda/blob/main/tests/sequential/disruption/disruption_test.go#L155)), thanks to running sequentially, this will be the only test in parallel, not affecting to other tests in progress if you kill the pods or do whatever you need

Reply:
Actually after applying the wrongScaledObject the errors counter value continuously increasing... as the pollingInterval is 2 so if I sleep the program for 2 seconds the countervalue increased by 1 from previous value. So I was asserting that each time I apply the wrongscaledObject the errors are incremented. And the errors couterValues always contain the previous test run's counterValues so if I want to assert that it create same amount of errors each time or not then I think I have to restart the keda-operator every time when I need to fetch the metrics. In the last github action run I saw that the 'keda_scaler_errors' metrics counterValue is 0, but in my local cluster it works properly. Can you please check the trigger section of wrongScaledObjectTemplate.

Reply:
In that case, don't use `Greater`, use `GreaterOrEqual` because the reported error is that `"22" is not greater than "22"`, maybe the different available resources is doing the difference

Reply:
Ok i will change that but keda_scaler_errors shouldn't be zero . In the second last github action the errors message was "***0" is not greater than "***0" . That's confusing.

Reply:
yes... sadly GH Action masks too much info, in the future if you need to read the unmasked text, we added some time ago an artifact generation with the logs from tests executions:

![image](https://user-images.githubusercontent.com/36899226/231963455-30b9302d-ffb8-4caf-9c4f-45e37d9cd8cb.png)

There you can download a zip with all the test logs